commons_ocollection/ograph_extended.ml

```diff
@@ -318,7 +323,7 @@ let launch_gv_cmd filename =
   ()
 
 let display_graph_cmd =
```

Comment:
lazy will not save any resource. It will always be forced each time the ocaml program starts.
Maybe you also want to add a () parameter to display_graph_cmd so it's evaluated only when we actually need to launch gv.

Reply:
should be fine now. They're now all normal function calls, with laziness under the hood.